runtime/gc_base/StackSlotValidator.cpp

```diff
@@ -45,10 +45,21 @@ MM_StackSlotValidator::reportStackSlot(MM_EnvironmentBase* env, const char* mess
 	J9VMThread* walkThread = _walkState->walkThread;
 	Trc_MM_StackSlotValidator_reportStackSlot_Entry(env->getLanguageVMThread(), walkThread);
 
-	char* threadName = getOMRVMThreadName(walkThread->omrVMThread);
-	j9tty_printf(PORTLIB, "%p: %s in thread %s\n", walkThread, message, NULL == threadName ? "NULL" : threadName);
-	Trc_MM_StackSlotValidator_thread(env->getLanguageVMThread(), message, NULL == threadName ? "NULL" : threadName);
-	
+	char* threadName = NULL;
+	if (NULL != walkThread->omrVMThread) {
```

Comment:
wondering if it would be cleaner to make the determination here, and use sprintf(threadName, "Contination@%p") instead, where threadName would be explicitely malloced (and later freed) for this case (could be even put into a helper get/releaseContinuationThreadName)

the error print and trace point would could be shared (with or without 'thread' in the print) like
`j9tty_printf(PORTLIB, "%p: %s in %s\n", walkThread, message, threadName);`
or 
`j9tty_printf(PORTLIB, "%p: %s in thread %s\n", walkThread, message, threadName);`



@dmitripivkine opinion?